server/storage/sql_models.go

```diff
@@ -14,6 +14,7 @@ type TUFFile struct {
 
 // TableName sets a specific table name for TUFFile
 func (g TUFFile) TableName() string {
+	// NOTE: if this value changes, please also change it in SQLStorage.Delete
```

Comment:
Unfortunately we didn't want to string-interpolate an SQL statement.  :|  Which  means hard-coding it in the DELETE string.  Probably the right way to fix this is to submit a patch to GORM upstream to enable hard-delete?  

Although there's this `scope.Search.Unscoped` check in the delete function that I don't really know how to use:  https://github.com/jinzhu/gorm/blob/master/callback_delete.go#L29 - maybe that does something interesting.